### DISCUSSION TITLE
Welcome to sd-webui-openpose-editor Discussions!

### DISCUSSION THREAD

**Discussion by huchenlei** [huchenlei/sd-webui-openpose-editor#16] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Jackyutopia** [comment]
你好 测试了很多次 重装了controlnet和openpose editor 但是依旧报错：Error：Malformed frame message received: [object Object]，骨骼图无法显示，需要怎么处理呢 谢谢

**Comment by Jackyutopia** [comment]
已经第三天了 依旧没法解决 dist.zip腋重新覆盖了 还是没有解决问题 可以有解决方案吗 谢谢

**Comment by huchenlei** [comment]
你看下是不是这个issue里说明的问题https://github.com/huchenlei/sd-webui-openpose-editor/issues/32 。视频里是关了modal重新点edit就好了

**Comment by gaihk** [comment]
![image](https://github.com/huchenlei/sd-webui-openpose-editor/assets/138961719/7017bac3-5494-4b65-9136-c0ade795a733)
老师您好，这个插件非常好，能解决很多问题。可惜，我还没尝到。。。我的EDIT在左边的。。按了下去没反应也没有窗口弹出来，按JSON却可以下载JSON。 希望老师能解决我的问题，感谢指导！

**Reply by huchenlei** [reply]
这个问题其他人也遇到过，看#33。最后也没找到核心cause，不过似乎重装一些环境能解决这个问题

**Reply by gaihk** [reply]
谢谢老师的回复，我看了#33的贴一遍，我重装了SD和CONTROL NET，换浏览器可惜问题还在。我打算搁置一下，等您的下一次更新。
所以，我另外单独打开它做JSON，也保存了JSON。可是，SD内置的OPEN POSE导入老师的JSON说invalid json. 学生想请教，哪里能用上老师的程序导出的JSON？ 感谢！

**Reply by huchenlei** [reply]
这个症状是css没起作用外加close按钮也不管用（js整个都没注册上），我怀疑是你的gradio版本的问题或者是webui版本的问题。你把webui和gradio升级到最新版试试。其他editor对open pose json的支持都是他们自己实现的，有的支持有的不支持。看你用的是哪个插件了

**Reply by gaihk** [reply]
谢谢老师，我搞好了。我装上秋叶版本的SD，完美用上老师的插件。再次感谢您的无私分享，非常好用。

**Comment by wangzhiguang1990** [comment]
请问我重新装了很多次插件，也重新覆盖了那个文件夹，但是用了openpose处理器后，骨骼图那里始终看不到edit按钮，用的是秋叶的一键部署

**Reply by huchenlei** [reply]
看不到`Edit`请确认ControlNet的版本升级到了1.1.216以上。

**Comment by drinkgin818** [comment]
Is it possible to get bones for the toes to help with the generation of feet?

**Comment by AlbertNeyro** [comment]
Is it possible to make the Extension appear in the SD-webui as a tab? Would save some time for me.

<img width="926" height="122" alt="image" src="https://github.com/user-attachments/assets/ff01930a-7d97-4195-9470-d7a8237d3397" />
